krtc/cmpauth.py

```diff
@@ -0,0 +1,119 @@
+import os
+import logging
+import re
+import argparse
+from urllib.parse import urlparse
+import getpass
+import requests
```

Comment:
`requests` isn't listed as a requirement, so the test suite is failing; please add it to `requirements.txt` and `conda-recipe/meta.yaml` (under the `requirements/run` heading)